.travis.yml

```diff
@@ -2,3 +2,5 @@ language: node_js
 
 node_js:
   - '6'
+  - '4'
+  - '0.10'
```

Comment:
Sorry for that. Node 0.10 [will not be LTS by the end of this month](https://github.com/nodejs/LTS#lts-schedule), so there is no need to complicate even further this project. If you prefer to maintain a fork, I can understand that.